frontend/public/components/overview/deployment-config-overview.tsx

```diff
@@ -15,12 +14,23 @@ import {
 import { OverviewDetailsResourcesTab } from './resource-overview-page';
 import { OverviewItem } from '.';
 import { ResourceOverviewDetails } from './resource-overview-details';
+import PodRingDataContoller from '../../../packages/dev-console/src/components/topology/PodRingDataController';
+import PodsRing from './../../../packages/dev-console/src/components/PodsRing';
 
 const DeploymentConfigOverviewDetails: React.SFC<DeploymentConfigOverviewDetailsProps> = ({item: {obj: dc}}) => {
   return <div className="overview__sidebar-pane-body resource-overview__body">
     {dc.spec.paused && <WorkloadPausedAlert obj={dc} model={DeploymentConfigModel} />}
     <div className="resource-overview__pod-counts">
-      <DeploymentPodCounts resource={dc} resourceKind={DeploymentConfigModel} />
+      <PodRingDataContoller
+        namespace={dc.metadata.namespace}
+        knative={false}
```

Comment:
here `export const DeploymentConfigOverviewPage: React.SFC<DeploymentConfigOverviewProps> = ({item})` `item` has all resources data, no need to put false